Sito/php/dbaccess.php

```diff
@@ -99,7 +123,7 @@ function checkMinLen($string) {
     //Controlla che la stringa non contenga caratteri speciali
     function checkAlfanumerico($string) {
         if(!checkMinLen($string)) return false;
-        if (!preg_match('/^[a-zA-Z0-9]+$/', $string)) {
+        if (!preg_match('/^[a-zA-Z0-9 .,]+$/', $string)) {
```

Comment:
Io non cambierei questa funzione qui perchè ad esempio io l'ho usata per il nome utente perchè non deve contenere caratteri speciali e magari l'ha usata qualcun'altro. Se ti serve per il titolo creane un'altra. Ti sottolineo però che questa continua a non permettere di mettere gli spazi tra le parole e quindi dove la usi tu per i titoli si può usare un'unica parola attaccata e basta.

Reply:
ok, ora ne creo una diversa.
A me sembrava accettasse gli spazi, i punti e le virgole

Reply:
![Schermata 2019-12-23 alle 22 20 36](https://user-images.githubusercontent.com/40260964/71381101-75ec4600-25d2-11ea-9d48-e98018789216.png)

Reply:
Però in effetti non mi valida gli accenti sul sito

Reply:
Ho lasciato il testo sotto vuoto per far si che non me lo salvasse
ho tolto gli accenti e ho lasciato solo spazio, punto, virgola, ? e !

![Schermata 2019-12-23 alle 22 29 57](https://user-images.githubusercontent.com/40260964/71381418-c2845100-25d3-11ea-9d71-e17b8c3e109c.png)

Reply:
Ok, io avrei tolto direttamente il controllo perchè magari servono anche i : e poi magari uno vuole scrivere un nome in spagnolo e gli serve una lettera particolare ma chissene basta che uno riesca a scrivere quello che è già nel database sennò è una presa in giro xD